### PR TITLE
istioctl verify-install: Colorize success/failure markers

### PR DESCRIPTION
**Pull request by esnible**
Resolves https://github.com/istio/istio/issues/29336

The only output change is to colorize the little ✔ and ✘ markers.

Uses the same technique as `istioctl analyze` to see if it is safe to colorize.

I didn't test this on Windows.  Windows has some non-ANSI terminals.  Our progress bar uses github.com/mattn/go-colorable for those Windows terminals.  I didn't bring that it, because I don't have a Windows machine to test on and invoking it directly outside of a progress bar would mean changing _go.mod_.

### CODE REVIEW

File: istioctl/pkg/verifier/verifier.go [hunk 1]
```diff
@@ -78,9 +81,16 @@ func NewStatusVerifier(istioNamespace, manifestsPath, kubeconfig, context string
 		kubeconfig:       kubeconfig,
 		context:          context,
 		iop:              installedIOP,
+		successMarker:    "✔",
+		failureMarker:    "✘",
 	}
 }
 
+func (v *StatusVerifier) Colorize() {
+	v.successMarker = color.New(color.FgGreen).Sprint("✔")
```
Comment (shamsher31) [inline istioctl/pkg/verifier/verifier.go:90]: nit:
```suggestion
	v.successMarker = color.New(color.FgGreen).Sprint(v.successMarker)
```

Comment (esnible) [reply istioctl/pkg/verifier/verifier.go]: :+1: done

**Review by howardjohn — APPROVED** [review]

**Review by johnma14 — APPROVED** [review]

**Review by shamsher31 — APPROVED** [review]
Thanks.

**Pull request merged**